Dalamud/Game/Gui/Nameplates/Model/INameplateNode.cs

```diff
@@ -0,0 +1,30 @@
+using Dalamud.Game.Text.SeStringHandling;
+
+namespace Dalamud.Game.Gui.Nameplates.Model;
+
+/// <summary>
+/// Represents a nameplate text node.
+/// </summary>
+public interface INameplateNode
```

Comment:
I wonder if it's better to just call this NameplateUpdateString or NameplateUpdateArgument or something else which doesn't use "node". When I think "node" in terms of the addons like NamePlate I think of the persistent UI nodes which make up the plates.

Reply:
I used "Element" before. But then I used "Node" as it the potential to match future expansion of the code.

Reply:
I see. I think if actual UI nodes are ever added to this there will likely need to be new field/event/etc. types, so I'm not sure there is really any forwards-compatibility to calling these "Nodes" here. If not "arguments", maybe "elements"? Things like prefix, name and free company are actually concatenated into the final name string and aren't even separate nodes in the final plate.